packages/contracts-bedrock/test/libraries/Bytes.t.sol

```diff
@@ -145,6 +148,9 @@ contract Bytes_slice_TestFail is Test {
     /// @notice Tests that, when given a start index `n` that is greater than
     ///         `type(uint256).max - n`, the `slice` function reverts.
     function testFuzz_slice_rangeOverflows_reverts(bytes memory _input, uint256 _start, uint256 _length) public {
+        _length = bound(_length, 0, _input.length);
+        _start = bound(_start, type(uint256).max - _length, type(uint256).max);
```

Comment:
Since `bound` ranges are inclusive, to match the assume conditions that were here before I think this needs to be
```suggestion
        _length = bound(_length, 0, _input.length - 1);
        _start = bound(_start, type(uint256).max - _length + 1, type(uint256).max);
```

Reply:
I did specifically not subtract 1 because it will panic when the input length is 0

Reply:
Ok so if we want to remove the `assume` calls, then we can do a `if length == 0 then 0 else length - 1` ?

Reply:
Your bounds are looser than the `assume` bounds, and this avoids needing a conditional like `_input.length == 0 ? 0 : _input.length - 1`. So since we still have the `assume` statements below to catch the looser bounds, this is ok. I would suggest just adding a comment about this, since from the existing comments it's not clear which ranges (the bound ones vs. assume ones) are the desired ones

Reply:
This is a great comment, I will update